packages/stargate/src/aminotypes.spec.ts

```diff
@@ -12,12 +12,22 @@ import {
   MsgSend,
   MsgUndelegate,
 } from "@cosmjs/launchpad";
+import {
+  MsgFundCommunityPool,
+  MsgSetWithdrawAddress,
+  MsgWithdrawDelegatorReward,
+  MsgWithdrawValidatorCommission,
+} from "@cosmjs/launchpad/types/msgs";
```

Comment:
Can we import them from `@cosmjs/launchpad` as above?

Reply:
Oops, thanks